andrea/css/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+    background: rgb(221,133,92);
+    background: linear-gradient(45deg, rgba(221,133,92,1) 0%, rgba(72,108,104,1) 100%, rgba(73,108,104,1) 100%);
+    height: 100vh;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 300;
+}
+
+
+button {
+    border: none;
+    text-transform: capitalize;
+    font-weight: 300;
+    padding: 8px;
+    width: 132px;
+    height: 40px;
+    margin: 16px 8px;
+    border-radius: 8px;
+}
+
+header {
+    display: flex;
+}
+
+.navbar {
+    display: flex;
+    width: 100%;
+    align-items: center;
+}
+
+.logotype {
+    margin-right: auto;
+}
+.card-img {
+    display: flex;
+    position: absolute;  
+    height: 100vh;
+    top: 0;
+    justify-content: center;
+    right: 200px; 
+    z-index: -1; 
+}
+
+#sign-btn {
+    background-color:  #F8ECDC;
+    color: #2E4F48;
+}
+
+#login-btn {
+    background-color: #2E4F48;
+    color: #F8ECDC;
+    margin-right: 32px;
+}
+
+#sign-btn:hover, #login-btn:hover {
+    background-color: rgba(221,133,92,1);
+}
+.fab {
```

Comment:
what does `fab` stands for?

Reply:
It's the class of the font awesome icons but I will add my own custom class :)